### ISSUE TITLE
Overfitting of RNN models: weight decay

### ISSUE THREAD

**Issue by mi92** [BorgwardtLab/multicenter-sepsis#22]
we moved to per patient loss to per time point loss, and samples weights were determined on a timepoint basis.
